[Bug][Testing] AST parsing fails on torch `sqrt`

### Bug Explanation

A number of open PRs are being affected by AST's failure to parse the `sqrt` torch frontend function. The CI fails with the following error message ((example job)[https://github.com/unifyai/ivy/actions/runs/3853660784/jobs/6566851736]):

```
ivy/functional/ivy/data_type.py:80: in _get_function_list
(https://github.com/unifyai/ivy/actions/runs/3853660784/jobs/6566851736#step:6:73)
    tree = ast.parse(_lstrip_lines(inspect.getsource(func)))
(https://github.com/unifyai/ivy/actions/runs/3853660784/jobs/6566851736#step:6:74)
/usr/lib/python3.8/ast.py:47: in parse
(https://github.com/unifyai/ivy/actions/runs/3853660784/jobs/6566851736#step:6:75)
    return compile(source, filename, mode, flags,
(https://github.com/unifyai/ivy/actions/runs/3853660784/jobs/6566851736#step:6:76)
E     File "<unknown>", line 10
(https://github.com/unifyai/ivy/actions/runs/3853660784/jobs/6566851736#step:6:77)
E       def sqrt(self):
(https://github.com/unifyai/ivy/actions/runs/3853660784/jobs/6566851736#step:6:78)
E       ^
(https://github.com/unifyai/ivy/actions/runs/3853660784/jobs/6566851736#step:6:79)
E   IndentationError: unexpected indent
```

### Steps to Reproduce Bug

CI logs. I was also able to reproduce the error locally on master via

```
pytest ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py -k 'test_torch_instance_sqrt'
```

It's worth noting that invoking `pytest` with some other test in the directory (instead of `test_torch_instance_sqrt`) also fails.

### Environment

N/A

### Ivy Version

master

### Backend

- [ ] NumPy
- [ ] TensorFlow
- [X] PyTorch
- [ ] JAX

### Device

_No response_

--- a/ivy/functional/frontends/torch/tensor.py
+++ b/ivy/functional/frontends/torch/tensor.py
@@ -492,15 +492,7 @@ class Tensor:
             return torch_frontend.tensor(ivy.array(self._ivy_array).full_like(max))
         return torch_frontend.clamp(self._ivy_array, min=min, max=max, out=out)
 
-    @with_unsupported_dtypes(
-        {
-            "1.11.0 and below": (
-                "float16",
-                "bfloat16",
-            )
-        },
-        "torch",
-    )
+    @with_unsupported_dtypes({"1.11.0 and below": ("float16", "bfloat16")}, "torch")
     def sqrt(self):
         return torch_frontend.sqrt(self._ivy_array)
 

